hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -689,726 +689,727 @@
                   '''
-        count:    "NumManufStateWords", // 8 x 32bit = 256bit
         swaccess: "ro",
         hwaccess: "hwo",
         hwext:    "true",
-        cname:    "WORD",
-        tags: [ // Life cycle internal HW will update status so can not auto-predict its value.
+        tags: [ // The value of this register is fixed and determined by HW.
                 "excl:CsrAllTests:CsrExclCheck"],
         fields: [
-          { bits: "31:0"
+          { bits: "RevisionIdWidth-1:0",
+            name: "REVISION_ID"
+            desc: '''
+                  Product revision ID. Assigned by the Silicon Creator.
+                  The encoding is not specified other than that different tapeouts must be assigned different revision numbers.
+                  I.e., each base or metal layer respin must be reflected so that software can rely on it to modify firmware and driver behavior.
+                  Zero is an invalid value.
+                  '''
+          }
+          { bits: "31:RevisionIdWidth",
+            name: "RESERVED"
+            resval: 0
+            desc: '''
+                  Reserved bits.
+                  Set to zero.
+                  '''
           }
         ]
       }
-    }
-  ]
+
+      ///////////////
+      // Device ID //
+      ///////////////
+      { multireg: {
+          name:     "DEVICE_ID",
+          desc:     '''
+                    This is the 256bit DEVICE_ID value that is stored in the HW_CFG0 partition in OTP.
+                    If this register reads all-one, the HW_CFG0 partition has not been initialized yet or is in error state.
+                    If this register reads all-zero, this is indicative that the value has not been programmed to OTP yet.
+                    '''
+          count:    "NumDeviceIdWords", // 8 x 32bit = 256bit
+          swaccess: "ro",
+          hwaccess: "hwo",
+          hwext:    "true",
+          cname:    "WORD",
+          tags: [ // Life cycle internal HW will update status so can not auto-predict its value.
+                  "excl:CsrAllTests:CsrExclCheck"],
+          fields: [
+            { bits: "31:0"
+            }
+          ]
+        }
+      }
+      //////////////////////////////
+      // Manufacturing State Bits //
+      //////////////////////////////
+      { multireg: {
+          name:     "MANUF_STATE",
+          desc:     '''
+                    This is a 256bit field used for keeping track of the manufacturing state.
+                    '''
+          count:    "NumManufStateWords", // 8 x 32bit = 256bit
+          swaccess: "ro",
+          hwaccess: "hwo",
+          hwext:    "true",
+          cname:    "WORD",
+          tags: [ // Life cycle internal HW will update status so can not auto-predict its value.
+                  "excl:CsrAllTests:CsrExclCheck"],
+          fields: [
+            { bits: "31:0"
+            }
+          ]
+        }
+      }
+    ]
+  }
 }
```

Comment:
Nit: I'm not sure the indentation is right here: the "registers: " line on line 692 hasn't moved horizontally but now the final closing brace is 2 characters to the left.

Reply:
You're completely right: I think I must have misread the diff. Sorry!